src/test/java/com/google/songgame/TitleFormatterTest.java

```diff
@@ -0,0 +1,128 @@
+package com.google.songgame.data;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.Ignore;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/*
+  https://music.youtube.com/playlist?list=RDCLAK5uy_kmPRjHDECIcuVwnKsx2Ng7fyNgFKWNJFs
+
+*/
+
+@RunWith(JUnit4.class)
+public final class TitleFormatterTest {
+
+  @Test
+  public void simpleCapitalization() {
+    TitleFormatter titleFormatter = new TitleFormatter();
```

Comment:
Advanced and optional suggestion:

You could create a private instance variable for titleFormatter and assign it in an `@Before` method, that way you don't have to create it at the beginning of each test.  See "Using Setup Methods" in this article:
https://www.testwithspring.com/lesson/introduction-to-junit-4-test-classes/

This is pretty advanced and optional, so feel free to skip, but I wanted to let you know about this.